sentry/resource_sentry_metric_alert.go

```diff
@@ -318,6 +323,7 @@ func expandMetricAlertTriggerActions(actionList []interface{}) []*sentry.MetricA
 			Type:             sentry.String(actionMap["type"].(string)),
 			TargetType:       sentry.String(actionMap["target_type"].(string)),
 			TargetIdentifier: sentry.String(actionMap["target_identifier"].(string)),
+			IntegrationID:    sentry.Int(actionMap["integration_id"].(int)),
```

Comment:
This needs to be conditionally applied because otherwise we will ask for Sentry to create an integration of id = 0, which won't exist. We will see errors like:

```txt
        Error: POST https://sentry.io/api/0/projects/canva-dev-2/tf-project-928553362600179404/alert-rules/: 400 map[integration:[Invalid pk "0" - object does not exist.]]
```

Thus, we need to conditionally add it via something like

```go
		if v, _ := actionMap["integration_id"].(int); v != 0 {
			action.IntegrationID = sentry.Int(v)
		}
```

You will also need to do the similar to `flattenMetricAlertTriggerActions`.